content/ember/v6/deprecate-import-alias-from-object-computed.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Deprecate import alias from @ember/object/computed
+until: 7.0.0
+since: 6.0.0
+---
+
+`alias` from `@ember/object/computed` is deprecated.
+
+It created a second property that reflected another property's value.
+
+## Migration
+Prefer direct access to the original tracked state, or use a simple getter and setter.
```

Comment:
`alias` could point to a computed property, a getter, or really anything so this may need to describe how make sure they work going forward.